charts/index.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+entries:
+  kubeflow-trainer:
+  - apiVersion: v2
+    created: "2025-03-29T07:12:05.068599096+05:30"
+    dependencies:
+    - condition: jobset.install
+      name: jobset
+      repository: oci://us-central1-docker.pkg.dev/k8s-staging-images/charts
+      version: v0.8.0
+    description: A Helm chart for deploying Kubeflow Trainer on Kubernetes.
+    digest: e9540df42128f5df016de0c15f087dd3f400f46f0fa136ae965a7fedd3b00e22
+    home: https://github.com/kubeflow/trainer
+    keywords:
+    - kubeflow trainer
+    maintainers:
+    - name: andreyvelich
+      url: https://github.com/andreyvelich
+    - name: ChenYi015
+      url: https://github.com/ChenYi015
+    - name: gaocegege
+      url: https://github.com/gaocegege
+    - name: Jeffwan
+      url: https://github.com/Jeffwan
+    - name: johnugeorge
+      url: https://github.com/johnugeorge
+    - name: tenzen-y
+      url: https://github.com/tenzen-y
+    - name: terrytangyuan
+      url: https://github.com/terrytangyuan
+    name: kubeflow-trainer
+    type: application
+    urls:
+    - https://kubeflow.github.io/trainer/kubeflow-trainer/kubeflow-trainer-2.0.0.tgz
+    version: 2.0.0
+generated: "2025-03-29T07:12:05.055998684+05:30"
```

Comment:
That file is generated based in a specific version. It seems like it should not be checked-in?